app/services/create_planned_disbursement.rb

```diff
@@ -36,11 +40,13 @@ def call(attributes: {})
     result
   end
 
-  private
-
-  def convert_and_assign_value(planned_disbursement, value)
+  private def convert_and_assign_value(planned_disbursement, value)
```

Comment:
I see you tend to change the way we define private methods from indicating `private` on a line of code, followed by a list of methods underneath that are understood to be private. You put `private def` on every method. I guess it's a matter of preference. Since I was taught the other way during my bootcamp, I'd like to have a different point of view on whether you think using `private def` is better/more readable, if you don't mind sharing your opinion?

Reply:
LOL! It's funny you say that! This is exactly what I said to the dev that I first saw this as well! 😄 

I used the single `private` approach before this project, we have both all over this code base, the dev using the single private explained their reasoning and I couldn't argue with it (with a larger number of private methods you can loose sight of which methods are private) and I made the decision to make a attempt at consistency across the code and settled on the multiple privates – I wish standardrb declared a rule for this so we don't have to think about it, but it doesn't and I am not sure if it can and just haven't had time to check!